assets/javascript/app.js

```diff
@@ -0,0 +1,225 @@
+        ,
+        text: "Tyrion Lannister and Sansa Stark's Wedding"
+        ,
+        text: "Joffrey Baratheon and Margaery Tyrell's Wedding"
+    },
+    correct: "Joffrey Baratheon and Margaery Tyrell's Wedding"
+}, {
+    text: "3. What is the meaning of the phrase 'Valar Dohaeris' ? ",
+    option: {
+        text: "All men must live"
+        ,
+        text: "All men must die"
+        ,
+        text: "All men must serve"
+        ,
+        text: "All men must kill"
+    },
+    correct: "All men must serve"
+}, {
+    text: "4. Who sits on the Iron Throne in the end?",
+    option: {
+        text: "Jon Snow/Aegon VI Targaryen"
+        ,
+        text: "Daenerys Targaryen"
+        ,
+        text: "Brandon Stark"
+        ,
+        text: "No One"
+    },
+    correct: "No One"
+}, {
+    text: "5. Who is 'The Prince That Was Promised' according to the show?",
+    option: {
+        text: "Jon Snow/Aegon VI Targaryen"
+        ,
+        text: "Daenerys Targaryen"
+
+        ,
+        text: "Arya Stark"
+        ,
+        text: "Jamie Lannister"
+    },
+    correct: "Arya Stark"
+}, {
+    text: "6. What is Hodor's real name?",
+    option: {
+        text: "Willace"
+        ,
+        text: "Wylis"
+        ,
+        text: "Wyllis"
+        ,
+        text: "Wallace"
+    },
+    correct: "Wylis"
+}, {
+    text: "7. Who made The Iron Throne?",
+    option: {
+        text: "Balerion The Dread"
+        ,
+        text: "Aegon I Targaryen"
+        ,
+        text: "Aerys II Targaryen/The Mad King"
+        ,
+        text: "Tywin Lannister"
+    },
+    correct: "Balerion The Dread"
+}, {
+    text: "8. Which of the following is not one of Jamie Lannister's alias",
+    option: {
+        text: "Kingslayer"
+        ,
+        text: "Kingkiller"
+
+        ,
+        text: "Lord of Casterly Rock"
+        ,
+        text: "Lord Commander of the Kingsguard"
+    },
+    correct: "Lord of Casterly Rock"
+}, {
+    text: "9. Which of the following houses extinct?",
+    option: {
+        text: "Lannisters"
+        ,
+        text: "Tyrells"
+        ,
+        text: "Targaryens"
+        ,
+        text: "Baratheons"
+    },
+    correct: "Tyrells"
+}, {
+    text: "10. Who is currently The King/Queen of the Seven Kingdom?",
+    option: {
+        text: "Bran Stark"
+        ,
+        text: "Jon Snow"
+        ,
+        text: "Daenerys Targaryen"
+        ,
+        text: "No One"
+    },
+    correct: "No One"
+}]
+
+// start
+function start() {
+    time = 120;
+    var currentTime = timeConverter(time);
+    $timer.text(currentTime);
+    intervalID = setInterval(countDown, 1000);
+}
+function countDown() {
+    time--;
+    var currentTime = timeConverter(time);
+    $timer.text(currentTime);
+    if (time === 0) {
+        endGame();
+    }
+}
+
+    
+
+
+function buildQuestions() {
+    for (var i = 0; i < questions.length; i++) {
+        var $question = buildQuestion(questions[i], i);
```

Comment:
You don't need to add a $ to the beginning of the variable unless that is how you want to name it.